python/lsst/jointcal/dataIds.py

```diff
@@ -85,7 +86,7 @@ def makeDataRefList(self, namespace):
                     visitRefs[visit].append(ref)
 
                     md = ref.get("calexp_md", immediate=True)
```

Comment:
These two lines can be replaced with `wcs = ref.get('calexp_wcs', immediate=True)`.

Reply:
Then the next line will have to call ref.get("calexp_bbox"). I worry it may be less efficient than reading the metadata and using it for both, but I'll do it. It's probably good enough and I'm being too lazy to profile it.

Reply:
As far as I'm aware, it's currently a wash because those specific-metadata getters read the whole header(s) to get the specific thing, and that's something that the butler can eventually do caching to speed up.

We really shouldn't be operating on `calexp_md` things if at possible.